src/infra/modules/sns/main.tf

```diff
@@ -0,0 +1,20 @@
+data "aws_ssm_parameter" "subscribers" {
+  name = "subscribers"
+}
+
+locals {
+  emails = split(",", data.aws_ssm_parameter.subscribers.value)
+}
+
+resource "aws_sns_topic" "alarms" {
+  name         = format("%s-alarms", var.aws_region)
```

Comment:
Instead of the regione you can pass the name so that we can reuse the module to create more topics.